tools/gitcount.ipynb

```diff
@@ -146,15 +146,28 @@
    "execution_count": null,
    "metadata": {
     "ExecuteTime": {
-     "end_time": "2020-10-22T21:36:04.464127Z",
-     "start_time": "2020-10-22T21:36:04.461553Z"
+     "end_time": "2021-09-19T21:40:58.557330Z",
+     "start_time": "2021-09-19T21:40:58.550183Z"
     }
    },
    "outputs": [],
    "source": [
     "# get __version__\n",
-    "import spaghetti\n",
-    "__version__ = spaghetti.__version__"
+    "run ../spaghetti/_version.py"
```

Comment:
How does this work? I get SyntaxError trying to run the cell and undefined `get_versions` below.

Reply:
I somehow deleted the leading `%` it should be:
```python
%run ../spaghetti/_version.py
__version__ = get_versions()["version"]
```
This declares the `get_versions` function then isolates the versioneer version number like it does in [`__init__`](https://github.com/pysal/spaghetti/blob/075619c8bc30f9227efe1f61fdeb242ebbbbb956/spaghetti/__init__.py#L11). Pushing up the fix now.

Reply:
Smart, I didn't know about `%run` magic.